time.go

```diff
@@ -43,7 +43,9 @@ func Now(location *time.Location) Time {
 	calendar := &cal.Calendar{
 		Name: "moov-io/base",
 	}
-	calendar.AddHoliday(us.Holidays...) // TODO(adam): check for more? June 19th?
+	holidays := us.Holidays
+	holidays = append(holidays, us.Juneteenth) // TODO: revert to just us.Holidays after https://github.com/rickar/cal/pull/93
```

Comment:
It looks like Juneteenth falls on a Sunday this year, and the following Monday (June 20) will be observed as a banking holiday.  We should add a test case for June 20th

Reply:
Banking holidays are observed during the week, but businesses don't take Monday off when the holiday is on Sunday.

Reply:
additional test case added in 200d7cd